niceman/resource/docker_container.py

```diff
@@ -194,22 +207,40 @@ def _execute_command(self, command, env=None, cwd=None):
             lgr.debug("exec#%i: %s", i, line.rstrip())
         return (out, self.client.exec_inspect(execute['Id'])['ExitCode'])
 
+        exit_code = self.client.exec_inspect(execute['Id'])['ExitCode']
+        if exit_code not in [0, None]:
```

Comment:
so it is just `if exit_code`?

Reply:
Just raising the exception if the command exit code comes back other than zero. Are there other scenarios I am not thinking of?